app/views/admin/loans/show.html.slim

```diff
@@ -9,9 +9,14 @@
 
 - content_for :scripts
   = javascript_include_tag "admin/forms/forms"
-  = javascript_include_tag "admin/timeline/select_all"
   = javascript_include_tag "admin/expand"
   = javascript_include_tag "admin/calendar/calendar"
+  
```

Comment:
Now that we have moved over to Backbone, many JS files will need to be converted to Backbone. And yes, let's talk about per-page JS.